admin/services/group.service.js

```diff
@@ -6,22 +6,44 @@ const pupilDataService = require('../services/data-access/pupil.data.service')
 const groupService = {}
 
 groupService.getGroups = async function (query) {
-  return groupDataService.fetchGroups(query)
+  return groupDataService.getGroups(query)
 }
 
-groupService.getPupils = async function (schoolId) {
+groupService.getPupils = async function (schoolId, groupIdToExclude) {
   if (!schoolId) { return false }
-  return pupilDataService.getSortedPupils(schoolId)
+  let query = {}
```

Comment:
this should be done in the data service when we move it to SQL.
Can you just add a `// HACK` comment to this effect.
Once we have got near the end of the data layer refactor i will scan the sources for all `WARN`, `HACK` and `TODO`

Reply:
@GuyHarwood I've commented out the 'collation' line and added a TODO comment. I've also explained in the PBI what this means in terms of functionality.